webapp/graphite/render/functions.py

```diff
@@ -4049,6 +4050,39 @@ def randomWalkFunction(requestContext, name, step=60):
             int(epoch(requestContext["endTime"])),
             step, values)]
 
+def seriesByTag(requestContext, *tagExpressions):
```

Comment:
Personally I'd rather expect a list of `tagExpressions` instead of an undefined number of arguments which are then put into a list here. The call site can easily build that list out of the strings if it doesn't already have them as a such (which it probably does) and this will leave us with more flexibility to add other parameters in the future.